src/nvim/state.c

```diff
@@ -62,6 +64,17 @@ void state_enter(VimState *s)
       }
       // Flush screen updates before blocking
       ui_flush();
+
+      // Skip everything before the first screen update to prevent closing
+      // the startup log file when prompt() is called in VimEnter autocommand.
+      if (time_fd != NULL && first_redraw) {
+        TIME_MSG("ready for user input");
+        TIME_MSG("--- NVIM STARTED ---");
```

Comment:
why do we need both messages? `--- NVIM STARTED ---` has been redefined to mean "ready for user input" in this PR. So we don't need the "ready for user input" message?